src/apiV2/client.ts

```diff
@@ -85,15 +85,15 @@ export const streamQueryV2 = async ({
   const body: Query.Body = {
     query,
     search: {
-      corpora: [
+      corpora: corpusKey.split(",").map((key) => (
         {
-          corpus_key: corpusKey,
-          metadata_filter: metadataFilter,
+          corpus_key: key,
+          metadata_filter: metadataFilter ? `doc.source = '${metadataFilter}'` : undefined,
```

Comment:
Why do we only want to apply `metadataFilter` to `doc.source`? My understanding is that this can be any string we'd put in console, e.g. `doc.FILTER_ATTRIBUTE='value'`.

So in this example, line 91 would evaluate to: `metadata_filter: "doc.source = 'doc.FILTER_ATTRIBUTE='value''"`

If updating metadata filter isn't necessary to enable queries to multiple corpora, I suggest we keep it the same. WDYT?